<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
   <item>
    <widget class="MapViewEdit" name="mapViewEdit" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="tileLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="setTileBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Set Tile</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="setFeasibleBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Set Feasible</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="TileViewSelect" name="tileViewSelect" native="true">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="loadMapBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Load Map</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="createMapBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Create Map</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="setEventBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Set Event</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="loadTileBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Load Tile</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="saveMapBtn">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Save Map</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="authorLbl">
       <property name="text">
        <string>By: Jack</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapViewEdit</class>
   <extends>QWidget</extends>
   <header>mapviewedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TileViewSelect</class>
   <extends>QWidget</extends>
   <header>tileviewselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>createMapBtn</tabstop>
  <tabstop>loadMapBtn</tabstop>
  <tabstop>setTileBtn</tabstop>
  <tabstop>setFeasibleBtn</tabstop>
  <tabstop>setEventBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
